ACE/ace/Sock_Connect.cpp

```diff
@@ -1473,21 +1473,32 @@ ip_check (int &ipvn_enabled, int pf)
     {
 
 #if defined (ACE_WIN32)
+      static int recursing = 0;
+      if (recursing) return recursing;
```

Comment:
```suggestion
      if (recursing) return 1;
```
same effect, but I think this is more clear to the reader